repository_service_tuf/helpers/api_client.py

```diff
@@ -272,7 +280,7 @@ def send_payload(
 def get_md_file(file_uri: str) -> Metadata:
     if file_uri.startswith("http"):
         console.print(f"Fetching file {file_uri}")
-        response = requests.get(file_uri)
+        response = requests.get(file_uri, timeout=300)
```

Comment:
Why do you decide to add this?

Reply:
I added timeouts to mitigate bandit's `B113: request_without_timeout`.